src/main/java/com/clubobsidian/dynamicgui/function/impl/SetMoveableFunction.java

```diff
@@ -0,0 +1,48 @@
+package com.clubobsidian.dynamicgui.function.impl;
+
+import com.clubobsidian.dynamicgui.entity.PlayerWrapper;
+import com.clubobsidian.dynamicgui.function.Function;
+import com.clubobsidian.dynamicgui.gui.FunctionOwner;
+import com.clubobsidian.dynamicgui.gui.Gui;
+import com.clubobsidian.dynamicgui.gui.Slot;
+
+public class SetMoveableFunction extends Function {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 453447798953153174L;
+
+	public SetMoveableFunction(String name) 
+	{
+		super(name);
+	}
+
+	@Override
+	public boolean function(PlayerWrapper<?> playerWrapper) 
+	{
+		if(this.getOwner() instanceof Slot)
+		{
+			Boolean value = Boolean.valueOf(this.getData());
+			if(value != null)
+			{
+				FunctionOwner owner = this.getOwner();
+				if(owner != null)
+				{
+					if(owner instanceof Slot)
```

Comment:
Codacy found an issue: [These nested if statements could be combined](https://app.codacy.com/manual/virustotalop/DynamicGui/pullRequest?prid=5724759)